### PR TITLE
Fix default style sheet location

### PR DESCRIPTION
**Pull request by pavpanchekha**
In earlier versions of the book, the default style sheet was stored in a field on `Browser` but it's now a global constant and not all the code got updated.

### CODE REVIEW

File: src/lab10.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def load(self, url, payload=None):
             except dukpy.JSRuntimeError as e:
                 print("Script", script, "crashed", e)
 
-        self.rules = self.default_style_sheet.copy()
+        self.rules = DEFAULT_STYLE_SHEET.copy()
```
Comment (chrishtr) [inline src/lab10.py:163]: Did lab10 just not work previously?

Comment (pavpanchekha) [reply src/lab10.py:163]: I think `lab10` was using the `default_style_sheet` field being created in `lab8`. So yes `lab8` was redundantly storing the default style sheet in _both_ places, and some labs were using one place while some were using the other.

File: src/lab8.py [hunk 2]
```diff
@@ -243,23 +243,14 @@ def paint_tree(layout_object, display_list):
 
 @wbetools.patch(Tab)
 class Tab:
-    def __init__(self, tab_height):
```
[the remark this answers is not in the document]
Comment (pavpanchekha) [reply src/lab8.py:246]: You're right.

File: src/lab8.py [hunk 3]
```diff
@@ -369,7 +360,6 @@ def __init__(self):
         self.tabs = []
         self.active_tab = None
         self.focus = None
-        self.address_bar = ""
```
Comment (chrishtr) [inline src/lab8.py:372]: why remove this?

Comment (pavpanchekha) [reply src/lab8.py:372]: I think it now lives in the Chrome

Comment (chrishtr) [reply src/lab8.py:372]: I guess this was just a bug?

**Review by chrishtr — APPROVED** [review]

File: src/lab8.py (hunk 3 above)
Comment (pavpanchekha) [reply src/lab8.py:372]: Sure… more like a harmless leftover from a refactor.

**Pull request merged**